webpack.base.config.js

```diff
@@ -250,6 +250,9 @@ module.exports = (env) => {
       'esper.js': 'esper'
     },
     plugins: [
+      new webpack.DefinePlugin({
+        "COCO_PRODUCT": JSON.stringify(product) // Has to stringify
```

Comment:
why do we need this?

Reply:
since after we merge the `core/utils` the frontend has no way to find out if it is coco or ozar. we need either pass product from server to frontend or pass it from webpack to build. while, i think in webpack make it easiler

Reply:
Haha, we pass COCO_PRODUCT while building in [Dockerfile](https://github.com/codecombat/codecombat-server/blob/master/Dockerfile#L20)

That's how `product` gets ozaria 😅 
Anyway, this works too as long it doesn't override COCO_PRODUCT where it shouldn't